src/core/manager.c

```diff
@@ -3391,7 +3391,7 @@ int manager_loop(Manager *m) {
                         continue;
 
                 /* Sleep for watchdog runtime wait time */
-                r = sd_event_run(m->event, watchdog_runtime_wait());
+                r = sd_event_run(m->event, watchdog_runtime_wait(2));
```

Comment:
can you prefix the argument with `/* divisor= */`, this is otherwise too opaque. 

(here and all other calls)